flow/render_status.py

```diff
@@ -0,0 +1,211 @@
+# make separate python class for render_status
+import mistune
+from .scheduling.base import JobStatus
+
+
+class _render_status:
+    """A class for rendering status in different format.
+
+    This class provides method and string output for rendering status output in different format,
+    currently supports: print in terminal, markdown and html format provided in string
+
+    """
+
+    def __init__(self):
+        self.markdown_output = None
+        self.terminal_output = None
+        self.html_output = None
+
+    def generate_markdown_output(self, template, context):
+        self.markdown_output = template.render(**context)
+
+    def generate_terminal_output(self, template, context):
+        self.terminal_output = template.render(**context)
+
+    def generate_html_output(self, template, context):
+        self.generate_markdown_output(template, context)
+        self.html_output = mistune.markdown(self.markdown_output)
+
+    def render(self, template, template_environment, context, file, detailed, expand,
+               unroll, compact, pretty, option):
+        """render the status in different format for print_status.
```

Comment:
Capitalize "render".

Reply:
Can you also make the other parameters consistently capitalized below? e.g. "user provided Jinja2 template" -> "User provided...".